PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Pull Request Template
+
+---
+## Protocol Information
+
+- **Name:**
+- **Twitter Link:**
+- **Website Link:**
+- **Github:**
+- **Developer Docs:**
```

Comment:
Is this provided in a URL?

Reply:
It seems "Developer Docs" isn't very intuitive for you. Should we change it to "Developer Docs Link"?
cc @zodahu

Reply:
@zodahu, what are your thoughts on this?

Reply:
I think keeping it the same is fine from the same rule for Twitter and Website.